com.github.petebuffon.yafc.desktop

```diff
@@ -0,0 +1,5 @@
+[Desktop Entry]
+Type=Application
+Name=YAFC
+Exec=YAFC
```

Comment:
It's missing "Icon", at least

Reply:
There is no Icon, is that an issue?

Reply:
really?

Yes it is an issue. It is MANDATORY. No icon but a desktop file, it's a bug.